webapp-django/staticfiles/index.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
```

Comment:
These JS files need license headers. You can crib them from the other JS files in the repo.

I should have included them in my initial PR, my bad.